### PR TITLE
Restyle account login/signup pages

### PR DESCRIPTION
**Pull request by by-co**
## Restyle account flow

* Replaced most custom classes with Tachyons
* Styled links & checkbox to be theme color
* Added help text

Files touched: [sign up, login, forgot password, resend confirmation emails] x [artists, patrons]
### Related PRs
#60 

### Screenshots
screen recording 2019-02-27 at 12 06 am

### CODE REVIEW

File: app/views/buyers/registrations/new.html.erb [hunk 1]
```diff
@@ -5,37 +5,33 @@
...
+      <div class="mb3">
+        <%= f.label :password_confirmation, class: "mb1" %>
+        <%= f.password_field :password_confirmation, class: "textinput w-100", autocomplete: "new-password" %>
       </div>
 
-      <br /><br /><br />
-
-      <div class="actions">
-        <%= f.submit "Sign up", class: "button" %>
+      <div class="flex justify-between">
```
Comment (emilyzhong) [inline app/views/buyers/registrations/new.html.erb]: [nit] maybe a tiny bit more margin between the fields + submission button? around `mt4`

File: app/views/buyers/registrations/new.html.erb [hunk 2]
```diff
@@ -5,37 +5,33 @@
...
+        <div class="flex flex-column justify-around">
```
Comment (emilyzhong) [inline app/views/buyers/registrations/new.html.erb]: Seems like all the  `<%= render "buyers/shared/links" %>` are wrapped with this — possibly include this line (+ closing tag) in the the buyers link partial?

File: app/views/artists/confirmations/new.html.erb [hunk 3]
```diff
@@ -1,23 +1,22 @@
...
+        <p class="texthelp"> artists.sfai.edu required </p>
       </div>
 
-      <br /><br /><br /><br />
-
-      <div class="actions">
-        <%= f.submit "Resend confirmation", class: "button" %>
+      <div class="flex justify-between">
+        <div class="flex flex-column justify-around">
+          <%= render "artists/shared/links" %>
```
Comment (emilyzhong) [inline app/views/artists/confirmations/new.html.erb]: Similar comment as for buyers/shared/links, adding the parent div to the partial

File: app/views/artists/registrations/new.html.erb [hunk 4]
```diff
@@ -5,31 +5,31 @@
     <%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
       <%= devise_error_messages! %>
 
-      <div class="field">
+      <div class="mb3">
         <%= f.label :email %>
-        <%= f.email_field :email, class: "field", autofocus: true, autocomplete: "email" %>
+        <%= f.email_field :email, class: "textinput w-100", autofocus: true, autocomplete: "email" %>
+        <p class="texthelp"> artists.sfai.edu required </p>
```
Comment (emilyzhong) [inline app/views/artists/registrations/new.html.erb]: [nit] `artists.sfai.edu email required`

File: app/views/buyers/registrations/new.html.erb (hunk 2 above)
Comment (by-co) [reply app/views/buyers/registrations/new.html.erb]: Hmm that's a good suggestion. I thought about it and it feels better to keep all the Tachyons classes / `div`s in one place haha

Comment (by-co) [reply app/views/buyers/registrations/new.html.erb]: Actually no ur right

File: app/views/buyers/registrations/new.html.erb (hunk 1 above)
Comment (by-co) [reply app/views/buyers/registrations/new.html.erb]: Added `mb4` to the last element since the margins don't like to stack

**Pull request merged**